pkg/cluster/internal/providers/docker/images.go

```diff
@@ -36,10 +36,7 @@ func ensureNodeImages(logger log.Logger, status *cli.Status, cfg *config.Cluster
 	// pull each required image
 	for _, image := range common.RequiredNodeImages(cfg).List() {
 		// prints user friendly message
-		friendlyImageName := image
-		if strings.Contains(image, "@sha256:") {
-			friendlyImageName = strings.Split(image, "@sha256:")[0]
-		}
+		friendlyImageName, image := sanitizeImage(image)
```

Comment:
why do we return 2 values, one overwritting `image`?

Reply:
https://github.com/kubernetes-sigs/kind/blob/460c00b8aa035c49e790541f923d117b7026e0ac/pkg/cluster/internal/providers/podman/images.go#L84-L85

For docker it's not that useful, but for podman all 3 images are different. Added it to both to have the same flow.
We don't _need_ to overwrite `image`  here, but /shrug reused the same variable instead of defining a new one.